make/tools.mk

```diff
@@ -89,11 +88,11 @@ INSTALL            ?= $(X_INSTALL_TOOL)
 # Patch flags and tools for (cross) build
 FLAG_RELRO          = -Wl,-z,relro,-z,now
 FLAG_STDLIB         = 
-CFLAGS_EXT          = $(ARCHITECTURE_CFLAGS)
-CXXFLAGS_EXT        = $(ARCHITECTURE_CFLAGS)
-EXE_FLAGS_EXT       = $(ARCHITECTURE_CFLAGS)
-SO_FLAGS_EXT        = $(ARCHITECTURE_CFLAGS)
-LDFLAGS_EXT         = $(ARCHITECTURE_LDFLAGS)
+CFLAGS_EXT          = 
```

Comment:
I think we can use this variable as `NOARCH_CFLAGS` or get rid of this variable and `*_EXT` variables below.